specs/altair/p2p-interface.md

```diff
@@ -0,0 +1,114 @@
+| `sync_committee_{subnet_id}` | `SyncCommitteeSignature` |
+
+Definitions of these new types can be found in the [Altair validator guide](./validator.md#containers).
+
+#### Global topics
+
+Altair changes the type of the global beacon block topic and adds one global topic to propagate partially aggregated sync committee signatures to all potential proposers of beacon blocks.
+
+##### `beacon_block`
+
+The existing specification for this topic do not change from the Phase 0 document but the type of the payload does change to the (modified) `SignedBeaconBlock`. This type changes via the inclusion of the inner `BeaconBlockBody` that is modified in Altair. See the [state transition document](./beacon-chain.md#beaconblockbody) for Altair for further details.
+
+##### `sync_committee_contribution_and_proof`
+ 
+This topic is used to propagate partially aggregated sync committee signatures to be included in future blocks.
+
+The following validations MUST pass before forwarding the `signed_contribution_and_proof` on the network; define `contribution_and_proof = signed_contribution_and_proof.message` and `contribution = contribution_and_proof.contribution` for convenience:
+
+- _[IGNORE]_ The contribution's slot is for the current slot, i.e. `contribution.slot == current_slot`.
+- _[IGNORE]_ The block being signed over (`contribution.beacon_block_root`) has been seen (via both gossip and non-gossip sources).
+- _[IGNORE]_ There has been no other valid sync committee contribution for the declared `slot` by the validator referenced by `contribution_and_proof.aggregator_index`.
+- _[REJECT]_ The aggregator's validator index is within the current sync committee --
+  i.e. `state.validators[aggregate_and_proof.aggregator_index].pubkey in state.current_sync_committee.pubkeys`.
+- _[REJECT]_ The aggregate signature is valid for the message `beacon_block_root` and aggregate pubkey derived from the participation info in `aggregate_bits`.
+- _[REJECT]_ `contribution_and_proof.selection_proof` selects the validator as an aggregator for the slot -- i.e. `is_sync_committee_aggregator(state, contribution.slot, contribution_and_proof.selection_proof)` returns `True`.
+- _[REJECT]_ The `contribution_and_proof.selection_proof` is a valid signature of the `contribution.slot` by the validator with index `contribution_and_proof.aggregator_index`.
+- _[REJECT]_ The aggregator signature, `signed_contribution_and_proof.signature`, is valid.
+
+#### Sync committee subnets
+
+Sync committee subnets are used to propagate unaggregated sync committee signatures to subsections of the network.
+
+##### `sync_committee_{subnet_id}`
+
+The `sync_committee_{subnet_id}` topics are used to propagate unaggregated sync committee signatures to the subnet `subnet_id` to be aggregated before being gossiped to the global `sync_committee_contribution_and_proof` topic.
+
+The following validations MUST pass before forwarding the `sync_committee_signature` on the network:
+
```

Comment:
need a validation condition for 0s in the bitfield outside of the sub-committee for this subnet

Reply:
no longer applies w/ newer aggregation scheme